library/src/main/java/io/github/boguszpawlowski/composecalendar/Calendar.kt

```diff
@@ -145,6 +147,48 @@ public fun StaticCalendar(
   )
 }
 
+/**
+ * [Calendar] implementation without any mechanism for the selection.
+ *
+ * Basic usage:
+ * ```
+ *  @Composable
+ *  fun MainScreen() {
+ *    WeekCalendar()
+ *  }
+ * ```
+ *
+ * @param modifier
+ * @param today current day, defaults to [LocalDate.now]
+ * @param calendarState state of the composable
+ * @param dayContent composable rendering the current day
+ * @param monthHeader header for showing the current month and controls for changing it
+ * @param monthContainer container composable for all the days in current month
+ */
+@Composable
+public fun WeekCalendar(
```

Comment:
IMO this should not be a separate function, but we could re-use the existing ones, and just pass WeekMode via one of the arguments. If we start making a new function for every use-case, soon it will become very hard to maintain.

Reply:
In order to make it work, we could change the state of the calendar to not depend on `CurrentMonth`, but on a abstract `TimeFrame`, which depending on the chosen mode (week/month) would become YearMonth / our implementation of Week (I think there is no representation of a week in java.time).